changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Subscriptions Core Changelog ***
 
+= 7.4.3 - 2024-xx-xx =
+Fix - Exclude switch orders from the order_needs_payment logic to prevent errors during customer switch requests that result in $0 checkout scenarios.
```

Comment:
```suggestion
Fix - Prevent errors during checkout when a customer is switching their subscription product and does not require payment.
```

Feel free to disregard this suggestion but I think "Exclude switch orders from the order_needs_payment logic" would be too technical for the average merchant.

Reply:
Yep good point. Thanks for the suggestion.